example/.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The existing .eslintrc file is a backport of the D8 rules for D7. I recently took a pass at adding support for both D7 and D8 rules, and encountered the same question/concern about trying to maintain a copy of the D8 rules file. 

Perhaps it would make the most sense to handle this in Gadget, and either copy in the D7 starter file OR download a copy of the D8 rules depending on the user's choice?